src/data/set/lattice.lean

```diff
@@ -1190,6 +1192,18 @@ begin
   simpa only [Union, supr_subtype'] using h
 end
 
+lemma image_Inter {f : α → β} (hf : bijective f) (s : ι → set α) :
+  f '' (⋂ i, s i) = ⋂ i, f '' s i :=
+begin
+  casesI is_empty_or_nonempty ι,
+  { simp_rw [Inter_of_empty, image_univ_of_surjective hf.surjective] },
+  { exact (hf.injective.inj_on _).image_Inter_eq }
+end
+
+lemma image_Inter₂ {f : α → β} (hf : bijective f) (s : Π i, κ i → set α) :
+  f '' (⋂ i j, s i j) = ⋂ i j, f '' s i j :=
+by simp_rw image_Inter hf
```

Comment:
These lemmas are described neither by the title nor the PR description; were they intended for a different PR?

Reply:
Sorry, I forgot I added them when I opened the PR, but yes I do mean to add them.